RxCocoa/iOS/UISearchBar+Rx.swift

```diff
@@ -47,6 +56,36 @@ extension UISearchBar {
         
         return ControlProperty(values: source, valueSink: bindingObserver)
     }
-}
-
+    
+    /**
+     Reactive wrapper for `searchBarSearchButtonClicked:` delegate event.
+     */
+    public var rx_searchButtonClicked: ControlEvent<UISearchBar> {
```

Comment:
We would want to have `Void` instead of `UISearchBar` since this models an event.